pkg/api/api.go

```diff
@@ -240,6 +251,35 @@ func requestPostageBatchId(r *http.Request) ([]byte, error) {
 	return nil, errInvalidPostageBatch
 }
 
+func (s *server) authHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This handler is returning responses in plain text, while other handlers are returning json-encoded responses. Is this what it intended?